### ISSUE TITLE
[CY-2516] Add official remark-lint plugins

### ISSUE THREAD

**Issue by gpotter2** [codacy/codacy-remark-lint#7]
Hello, first thanks for this wonderful tool !

* **I'm submitting a ...**
[x] feature request

* **Summary**

remark-lint has most of its rules packed within the npm package, but also a list of 12 (supported) external (but approved) rules:
https://github.com/remarkjs/remark-lint/blob/master/readme.md#list-of-external-rules

Those are mostly bigger / more specific rules.

* **Other information**

Thhose rules required to be installed with npm (this a config file isn’t enough :-/ ) Apart from that they are normal rules. Having them integrated would be extremly useful, as some are especially useful:

- remark-lint-no-dead-urls — Check that external links are alive
- remark-lint-code — Lint fenced code blocks by corresponding language tags, currently supporting ESLint
- remark-lint-books-links — Ensure links in lists of books follow a standard format

I could try to implement it, but It would require me to get familiar with the fork first.

Thanks again for your work,
Regards

**Comment by josemiguelmelo** [comment]
Hi,

First of all, thank you for your contribution and feature request. 🙂 
We checked the plugins mentioned on the official documentation and we now support the majority of them.

Unfortunately, some of the plugins (such as `remark-lint-no-dead-urls`) require internet connection and we couldn't add it for security reasons.
From the ones you asked in specific, we added support for `remark-lint-code` and `remark-lint-books-links`. 

Thanks,
